frontend/src/metabase/writeback/components/ActionCreator/ActionCreator.tsx

```diff
@@ -65,6 +70,8 @@ function ActionCreatorComponent({
   >(undefined);
   const [showSaveModal, setShowSaveModal] = useState(false);
 
+  const [showDataRef, setShowDataRef] = useState(false);
```

Comment:
That's subjective, but should we aim to use `is` / `should` / `are` / etc kind of naming for booleans? `showDataRef` looks like something that can be a function too